optimizely/notification/center.go

```diff
@@ -34,8 +34,10 @@ type DefaultCenter struct {
 // NewNotificationCenter returns a new notification center
 func NewNotificationCenter() *DefaultCenter {
 	decisionNotificationManager := NewAtomicManager()
+	projectConfigUpdateNotificationManager := NewAtomicManager()
```

Comment:
This should be generic like `projectConfigNotificationManager` as we can later have other notifications aside from the` Update`